src/pages/s/[classNumber]/teachers.tsx

```diff
@@ -202,7 +202,7 @@ export const getServerSideProps: GetServerSideProps = async ({
   params,
   req,
 }) => {
-  const redirect = await protectPageFor("admin", req);
+  const redirect = await protectPageFor("student", req);
```

Comment:
great

Reply:
Yep, not sure what happened there. Probably copy-paste.